### ISSUE TITLE
Access though internet

### ISSUE THREAD

**Issue by KZL1992** [haugene/docker-transmission-openvpn#457]
Hello !

I am trying to find a way to access my dock transmission-openvpn. Everything is working in local.
If it can help, I am using the following configuration.

I guess that my problem is the LOCAL_NETWORK option while I am trying to access the WebGUI. It is working in local but I would like to use it from outside also. (I opened port 9091 in my router)

```
docker run --cap-add=NET_ADMIN --device=/dev/net/tun -d \
--restart=always \
--log-driver json-file \
--log-opt max-size=10m \
-v /mnt/storage/transmission:/data \
-v /opt/transmission/client.ovpn:/etc/openvpn/custom/default.ovpn \
-v /opt/transmission/server.crt:/etc/openvpn/custom/server.crt \
-v /opt/transmission/pass.txt:/etc/openvpn/custom/pass.txt \
-e "OPENVPN_OPTS=--inactive 3600 --ping 10 --ping-exit 60" \
-e "OPENVPN_PROVIDER=CUSTOM" -e "OPENVPN_USERNAME=KZL__ROGBOB" -e "OPENVPN_PASSWORD=VSCOTTO67400" \
-e "TRANSMISSION_PORT_FORWARDING_ENABLED=true" \
-e "TRANSMISSION_RPC_AUTHENTICATION_REQUIRED=true" \
-e "TRANSMISSION_RPC_ENABLED=true" \
-e "TRANSMISSION_RPC_USERNAME=KZL__ROGBOB" \
-e "TRANSMISSION_RPC_PASSWORD=VSCOTTO67400" \
-e "TRANSMISSION_BLOCKLIST_ENABLED=true" \
-e "TRANSMISSION_BLOCKLIST_URL=http://john.bitsurge.net/public/biglist.p2p.gz" \
-e "TRANSMISSION_WEB_UI=combustion" \
-e LOCAL_NETWORK=192.168.93.0/24 \
--sysctl net.ipv6.conf.all.disable_ipv6=0 \
--name "transmission-openvpn" \
-p 9091:9091 haugene/transmission-openvpn:latest


```

**Comment by Celoxocis** [comment]
since transmission 2.93 you will need to set proper whitelisting. (changes due to DNS rebind attacks)
also i highly recommend to SSL your connection through an proxy such as NGinx.
follow [this](https://www.htpcguides.com/secure-nginx-reverse-proxy-with-lets-encrypt-on-ubuntu-16-04-lts/) and use this as a base for [transmission](https://www.htpcguides.com/configure-transmission-reverse-proxy-win-mac-linux/)

**Comment by haugene** [comment]
My last comment in #354 explains a bit about this topic - I think it will answer your question.
In #413 the proxy solved it and in #398 we discuss some SSL stuff as well.

**Comment by haugene** [comment]
I'm closing this as it's probably related to configuration and not an issue with the container itself.
If you can't get it working after reading what I've linked above, ask for help in the gitter chat instead: https://gitter.im/docker-transmission-openvpn
